### PR TITLE
Release notes: Add a feature description to 0.13.6

### PR DESCRIPTION
**Pull request by qingwusunny**
Release notes: Add a feature description to 0.13.6

### CODE REVIEW

**Comment by fedepaol** [comment]
Can you also add a section to the advanced l2 configuration?

**Comment by qingwusunny** [comment]
_advanced_l2_configuration has updated @fedepaol

**Comment by qingwusunny** [comment]
when will it be released?

File: website/content/configuration/_advanced_l2_configuration.md [hunk 1]
```diff
@@ -68,3 +68,39 @@ In this way, all the IPs coming from `first-pool` will be reacheable only via `N
 and `NodeB`, and only one of those node will be choosen to expose the IP.
 
 On the other hand, IPs coming from `second-pool` will be exposed always via `NodeC`.
+
+### Specify network interfaces that LB IP can be announced from
+
+In L2 mode, by default a metallb speaker will announce LB IP from all network interfaces of a node. We can use `interfaces` in `L2Advertisement` to select a subset of them.
```
Comment (fedepaol) [inline website/content/configuration/_advanced_l2_configuration.md:74]:
```suggestion
In L2 mode, by default a metallb speaker announces the Loadbalancer IP from all the network interfaces of a node. We can use `interfaces` in `L2Advertisement` to select a subset of them.
```

File: website/content/configuration/_advanced_l2_configuration.md [hunk 2]
```diff
@@ -68,3 +68,39 @@ In this way, all the IPs coming from `first-pool` will be reacheable only via `N
...
+  name: example
+spec:
+  ipAddressPools:
+  - third-pool
+  interfaces:
+  - eth0
+  - eth1
+```
+
+By this `L2Advertisement`, LB IPs from `third-pool` will be only announced from interfaces `eth0` and `eth1` of all nodes.
```
Comment (fedepaol) [inline website/content/configuration/_advanced_l2_configuration.md:89]:
```suggestion
This `L2Advertisement` will make MetalLB announce the Services associated to IPs from `third-pool` only from the interfaces `eth0` and `eth1` of all nodes.
```

File: website/content/configuration/_advanced_l2_configuration.md [hunk 3]
```diff
@@ -68,3 +68,39 @@ In this way, all the IPs coming from `first-pool` will be reacheable only via `N
...
+  ipAddressPools:
+  - fourth-pool
+  nodeSelectors:
+  - matchLabels:
+      kubernetes.io/hostname: NodeA
+  interfaces:
+  - eth3
+```
+
+`interfaces` can also be used together with `nodeSelectors`. IPs from `fourth-pool` will be announced only from `eth3` of `NodeA`.
```
Comment (fedepaol) [inline website/content/configuration/_advanced_l2_configuration.md:106]:
```suggestion
The `interfaces` selector can also be used together with `nodeSelectors`. In this example, the IPs from `fourth-pool` will be announced only from `eth3` of `NodeA`:
```

I'd flip this, making the statement before the example.

**Comment by fedepaol** [comment]
I would also add a couple of notes:

One is the same as https://github.com/metallb/metallb/blob/main/design/layer2-bind-interfaces.md#note-multiple-l2advertisements-are-combined-by-taking-the-union

The second one is something like:
```
The interface selector won't affect how MetalLB is choosing the leader for a given L2 IP. This means that if it elects a leader where the selected interface is not available, the service won't be announced. The cluster administrator is responsible to use the combination of interfaces selector and node selector to avoid the problem.
```

Check here https://github.com/metallb/metallb/blob/main/website/content/configuration/_advanced_ipaddresspool_config.md?plain=1#L46 on how to add the note

**Comment by fedepaol** [comment]
> when will it be released?

I'd say once we merge this, sometimes next week - the week after AT MOST.

File: website/content/configuration/_advanced_l2_configuration.md (hunk 3 above)
Comment (qingwusunny) [reply website/content/configuration/_advanced_l2_configuration.md]: ok

File: website/content/configuration/_advanced_l2_configuration.md (hunk 2 above)
Comment (qingwusunny) [reply website/content/configuration/_advanced_l2_configuration.md]: done

File: website/content/configuration/_advanced_l2_configuration.md (hunk 1 above)
Comment (qingwusunny) [reply website/content/configuration/_advanced_l2_configuration.md]: done

**Comment by qingwusunny** [comment]
> I would also add a couple of notes:
> 
> One is the same as https://github.com/metallb/metallb/blob/main/design/layer2-bind-interfaces.md#note-multiple-l2advertisements-are-combined-by-taking-the-union
> 
> The second one is something like:
> 
> ```
> The interface selector won't affect how MetalLB is choosing the leader for a given L2 IP. This means that if it elects a leader where the selected interface is not available, the service won't be announced. The cluster administrator is responsible to use the combination of interfaces selector and node selector to avoid the problem.
> ```
> 
> Check here https://github.com/metallb/metallb/blob/main/website/content/configuration/_advanced_ipaddresspool_config.md?plain=1#L46 on how to add the note

ok

**Comment by fedepaol** [comment]
Took the liberty to change a couple of sentences and the commit message to speed up the process.
LGTM

**Pull request merged**